Not sure what it's trying to do with tcl EXTERN which maps to dllimport, just remove it

--- sqlite-src-3490100/src/tclsqlite.c.orig	2025-05-04 09:52:12.520409400 +0200
+++ sqlite-src-3490100/src/tclsqlite.c	2025-05-04 09:52:11.147865900 +0200
@@ -3970,6 +3970,11 @@
 # define PACKAGE_VERSION SQLITE_VERSION
 #endif
 
+#ifdef EXTERN
+#undef EXTERN
+#define EXTERN
+#endif
+
 /*
 ** Initialize this module.
 **
